tsconfig.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Hey @Fabs please cleanup removing the comments.

Reply:
We decided to left the comments to help people understand what the settings mean. I have no strong opinion here. Why would you clean up?

Reply:
Would also like to have the comments, for readability reasons

Reply:
I disagree with the choice of leaving the comments especially because we can and should always use the official documentation.

otherwise, why don't we do that for all config files? `.dockerignore` `.babelrc` `.editorconfig` `.eslintignore` `.eslintrc` `.stylelintrc` `Dockerfile` `jenkinsfile` `cypress.json`

That being said, I'm ok if we decide to leave the comments.

Reply:
I am inclined to think like @weblancaster (siding with consistency). I started with no comments, I read the docs. But when I ran `tsc --init` it had comments and other team members proposed leaving then. My original intention was to remove, but I caved in 🙈

Reply:
@DanielMSchmidt, in the end I choose to remove the comments. Are you ok with that decision?